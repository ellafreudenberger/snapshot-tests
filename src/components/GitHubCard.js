import React from 'react';
import Card from 'react-bootstrap/Card';

const GitHubCard = () => {
  return (
    <Card>
      <Card.Img variant="top" src="public/photos/me.HEIC" alt="me" />
      <Card.Body>
        <Card.Title>ellafreudenberger</Card.Title>
        <Card.Text>
          A Software Engineer Student at NYU
        </Card.Text>
        <Card.Footer>Graduating December 2023</Card.Footer>
      </Card.Body>
    </Card>
  );
};

export default GitHubCard;
