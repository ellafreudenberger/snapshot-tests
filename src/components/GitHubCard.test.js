import React from 'react';
import renderer from 'react-test-renderer';
import GitHubCard from './GitHubCard';

test('GitHubCard snapshot', () => {
  const component = renderer.create(<GitHubCard />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

