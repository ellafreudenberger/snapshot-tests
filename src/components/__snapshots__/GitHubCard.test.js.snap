// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubCard snapshot 1`] = `
<div
  className="card"
>
  <img
    alt="me"
    className="card-img-top"
    src="public/photos/me.HEIC"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      ellafreudenberger
    </div>
    <p
      className="card-text"
    >
      A Software Engineer Student at NYU
    </p>
    <div
      className="card-footer"
    >
      Graduating December 2023
    </div>
  </div>
</div>
`;
